js/components/starRating.js

```diff
@@ -32,7 +32,11 @@ Fliplet.FormBuilder.field('starRating', {
     };
 
     if (this.required && !this.readonly) {
-      rules.value.required = window.validators.required;
+      rules.value.required = function(value) {
+        if (typeof value !== 'undefined' && value >= 0) {
```

Comment:
This would have been the same as:

```suggestion
        return typeof value !== 'undefined' && value >= 0;
```

[We discussed](https://weboo.atlassian.net/browse/XOR-554?focusedCommentId=37636) using `rules.value.required = function(value) { return value; };`. Why was your solution different?